testsuite/integration-arquillian/tests/base/src/test/resources/arquillian.xml

```diff
@@ -87,6 +87,11 @@
         <property name="chromeArguments">${js.chromeArguments}</property>
     </extension>
 
+    <extension qualifier="webdriver-phantomjsbrowser">
+        <property name="browser">${js.browser}</property>
```

Comment:
We should hardcode "phantomjs" here. Phantomjs should be Phantomjs no matter what. :)

Reply:
Fixed. Thanks!